app/src/main/res/layout/item_rate.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="62dp"
+    android:orientation="horizontal"
+    android:paddingStart="24dp"
+    android:paddingEnd="12dp">
+
+    <include
+        android:id="@+id/currency"
+        android:layout_height="wrap_content"
+        android:layout_width="wrap_content"
+        android:layout_centerVertical="true"
+        layout="@layout/layout_currency" />
+
+    <TextView
+        android:layout_toEndOf="@id/currency"
+        android:id="@+id/price"
+        android:layout_alignParentEnd="true"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="12dp"
+        android:gravity="end"
+        android:textColor="@color/white"
+        android:textSize="16sp"
+        android:textStyle="bold"
+        tools:text="$4 808,00" />
+
+    <TextView
+        android:layout_toEndOf="@id/currency"
+        android:layout_marginTop="4dp"
+        android:layout_below="@id/price"
+        android:layout_alignParentEnd="true"
+        android:id="@+id/percent_change"
```

Comment:
could you use just id="@id/percent_change" ?

Reply:
the attempt to use @id instead of @+id leads to a mistake

Android resource linking failed
C:\Users\Alex\AndroidStudioProjects\LoftCoin\app\src\main\res\layout\item_rate.xml:30: error: resource id/percent_change (aka com.alexandr4.loftcoin:id/percent_change) not found.
error: failed linking file resources.